firstvoices/backend/tests/test_search/test_query_params.py

```diff
@@ -152,3 +153,100 @@ def test_both(self):
         # Should also contain the full text search matches
         assert self.expected_multi_match_string in str(search_query)
         assert self.expected_match_full_text_search_string in str(search_query)
+
+
+@pytest.mark.django_db
+class TestCategory:
+    def setup(self):
+        self.site = factories.SiteFactory()
+        self.parent_category = factories.ParentCategoryFactory(site=self.site)
+        self.child_category = factories.ChildCategoryFactory(
+            parent=self.parent_category
+        )
+
+    def test_default(self):  # default case
+        # relates to: SearchQueryTest.java - CategoryIdTest.testDefault(),
+        # CategoryIdTest.test_category_blank_allowed()
+        search_query = get_search_query()
+        search_query = search_query.to_dict()
+
+        assert "categories" not in str(search_query)
+
+    def test_invalid_category(self):
```

Comment:
You can just remove this if it isn't testing anything. Maybe move this comment onto the TestCategory class level